### ISSUE TITLE
removes non-cooking recipes for silver/mithril ingots

### ISSUE THREAD

**Issue by programmerjake** [joe7575/techage#186]
when trying to removing the recipes for cooking to make silver/mithril ingots, the code instead removes all recipes that produce silver/mithril ingots, including recipes such as crafting a silver block into ingots:

https://github.com/joe7575/techage/blob/8183b05675c76c8da8677d2d934d345b918595c7/furnace/recipes.lua#L154-L155
https://github.com/joe7575/techage/blob/8183b05675c76c8da8677d2d934d345b918595c7/iron_age/recipes.lua#L141-L144

I think the fix would be something like changing the [`clear_craft`](https://api.luanti.org/core-namespace-reference/#gameplay) calls to not specify the output item and instead specify the inputs and type, that way it doesn't delete all the other recipes so you don't have to re-add them:
```lua
-- delete cooking iron lumps into steel ingots
minetest.clear_craft({
    type = "cooking",
    recipe = "default:iron_lump",
})
-- delete cooking silver lumps into silver ingots
minetest.clear_craft({
    type = "cooking",
    recipe = "moreores:silver_lump",
})
-- delete crafting buckets from steel ingots
minetest.register_craft({
    recipe = {
        {"default:steel_ingot", "", "default:steel_ingot"},
        {"", "default:steel_ingot", ""},
    }
})
```

**Comment by joe7575** [comment]
But that was the goal. It should not be possible to craft these items without the iron age furnace

**Comment by programmerjake** [comment]
yes, but i think you meant the goal to be that you only can't smelt lumps without the iron age furnace, not that you can't craft a silver block into ingots, or e.g. that if there's a mod that allows crafting armor back into ingots that with techage it no longer works because you inadvertantly deleted the recipes to craft silver armor into silver ingots.

**Comment by joe7575** [comment]
Thanks, I got it.
I changed the recipes for mithril, silver, steel, and bronze

**Comment by programmerjake** [comment]
thanks, though it would still break something like crafting bronze armor into bronze ingots because you specified `output` instead of specifying `recipe`

**Comment by joe7575** [comment]
Should be fixed now
